integration/src/ethereum/ethereum.ts

```diff
@@ -243,6 +243,52 @@ export function ethereumTests(get: () => { wallet: core.HDWallet; info: core.HDW
       TIMEOUT
     );
 
+    test(
+      "ethSignTx() - AVAX",
+      async () => {
+        if (!wallet) return;
+        // only test keepkey AVAX signing
+        // eslint-disable-next-line jest/no-jasmine-globals, jest/no-disabled-tests
+
+        const txToSign = {
+          addressNList: core.bip32ToAddressNList("m/44'/60'/0'/0/0"),
```

Comment:
use avalanche c chain coin type
```suggestion
          addressNList: core.bip32ToAddressNList("m/44'/9000'/0'/0/0"),
```

Reply:
👍